testing/containers/ceph/Dockerfile

```diff
@@ -14,9 +14,9 @@ RUN true && \
     yum update -y && \
     cv="$(rpm -q --queryformat '%{version}-%{release}' ceph-common)" && \
     yum install -y \
-        git wget curl make \
-        /usr/bin/cc /usr/bin/c++ \
-        "libcephfs-devel-${cv}" "librados-devel-${cv}" "librbd-devel-${cv}" && \
```

Comment:
I think these were indented differently because they're all arguments to the yum command, not commands of their own. If you prefer to keep it this way it doesn't bother me, but I wanted to point this out in case you thought the extra level of indent was a mistake.

Reply:
Yeah, the Dockerfile support in my IDE automatically reformatted that when I saved it, and I was too lazy to manually undo it. It doesn't bother me, but I can do if it bothers you.

Reply:
It's fine. If you don't change it I'll approve in ~24hrs if @anoopcs9 doesn't approve before me!